src/pages/ja/core-concepts/astro-components.md

```diff
@@ -153,17 +190,36 @@ const items = ["犬", "猫", "カモノハシ"];
 </ul>
 ```
 
-:::tip
-タグを動的に設定することもできます。
+AstroはJSXの論理演算子や三項演算子を用いて、HTMLを条件付きで表示することができます。
 
-```astro "El"
+```astro title="src/components/ConditionalHtml.astro" "visible"
 ---
-// src/pages/index.astro
-const El = 'div'
+const visible = true;
 ---
-<El>こんにちは！</El> <!-- <div>こんにちは！</div> としてレンダリングされます -->
+{visible && <p>私を見て！</p>}
+
+{visible ? <p>私を見て！</p> : <p>または私を見て！</p>}
+```
+
+### 動的なタグ
+
+HTMLタグ名、またはインポートしたコンポーネントを変数に設定することで動的なタグが利用できます。
+
+```astro title="src/components/DynamicTags.astro" /Element|(?<!My)Component/
+---
+import MyComponent from "./MyComponent.astro";
+const Element = 'div'
+const Component = MyComponent;
+---
+<Element>こんにちは！</Element> <!-- <div>こんにちは！</div> としてレンダリングされます -->
+<Component /> <!-- <MyComponent />としてレンダリングされます -->
 ```
-:::
+
+動的なタグを使用する場合：
+
+- **変数名をキャピタライズする** 例えば、`element`ではなくて`Element`を利用します。そうでなければ、Astroは変数名をリテラルなHTMLタグとしてレンダリングします。
+
+- **ハイドレーションのディレクティブはサポートしません.** [`client:*` ハイドレーションディレクティブ](/ja/core-concepts/framework-components/#hydrating-interactive-components)を利用する場合、Astroは本番用にバンドルするコンポーネントを知る必要がありますが、動的なタグパターンではこれが動作しません。
```

Comment:
Oops, one more link — not sure why I missed this before. I’ll commit this and get this merged 🥳 

```suggestion
- **ハイドレーションのディレクティブはサポートしません.** [`client:*` ハイドレーションディレクティブ](/ja/core-concepts/framework-components/#インタラクティブなコンポーネント)を利用する場合、Astroは本番用にバンドルするコンポーネントを知る必要がありますが、動的なタグパターンではこれが動作しません。
```